MIVisionX-setup.py

```diff
@@ -302,7 +302,7 @@ def ERROR_CHECK(call):
     pipNNEFversion = "protobuf==3.19.5" # TBD: NO NNEF Package for SLES
 
 pip3InferencePackagesRPM = [
-    'future==0.18.2',
```

Comment:
@hansely should be keep the RPM version unchanged as I am not sure if 1.0.0 is available on RHEL 8